wallet/build.gradle

```diff
@@ -100,7 +100,7 @@ android {
         disable 'MissingTranslation'
     }
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 19
```

Comment:
@HashEngineering what do you think about this change? I decided to switch from old API 16 to API 19 (which is still very old, though) when making system status and navigation bars translucent. In my opinion, we should systematically switch to newer API levels and redesign is a great opportunity for that.

Reply:
This will reduce the number of devices that can run the app.  We will need to see how many devices/users we would be excluding and run it by the Product Owner.  Bitcoin Wallet with their version 5.45 app had made their min SDK version = 19 (KitKat) and their version 7 (current) might be 21 or Android 5.0.  Users bring up issues concerning 4.4 and they are told to use a previous version of Bitcoin Wallet.  That works well enough given that Bitcoin doesn't have breaking protocol upgrades.  Previous versions of Bitcoin Wallet will work for the most part, while past versions of our app have lower likely hood of working.   

This particular change is probably OK, though, given that there probably are not many 4.1 to 4.3 devices around.  4.4 does have its own "cult" following though.

16 - Jelly Bean | 4.1
19 - Kit Kit | 4.4

Reply:
I was able to build this with 16.  I am not sure what features are made available by changing the min sdk level.  I thought that was only related to what devices that the app could run on.

Reply:
@tomasz-ludek I would like your comments on what I wrote above.  This might be a change that should be done separately with product owner's input.  This type of change could also be done repeatedly in the future and it would be a good idea to determine a process for this kind of change now.

Reply:
That is true, the code can be built with 16 and Gradle doesn't report any issues, but it doesn't mean the app will not crash on devices running Jelly Bean though...
I will test the app on Jelly Bean device and let's see what will happen.

Reply:
Crap, on Jelly Bean the app crashes when trying to initialize BLS:
`
Exception Ljava/lang/RuntimeException; thrown while initializing Lorg/dashj/bls/BLS;
11-21 11:59:06.851 2574-2574/hashengineering.darkcoin.wallet_test D/AndroidRuntime: Shutting down VM
`
It shows the onboarding screen but blow-up after clicking `Create new wallet`.

Reply:
That is a known issue.  We have had that problem with the current app since 6.6.2.

Therefore we should have 19 as our minimum on the redesign.

Reply:
This is a result of something in the BLS library and threads.